options/options.c

```diff
@@ -438,9 +438,14 @@ const struct m_sub_options filter_conf = {
     .opts = (const struct m_option[]){
         {"deinterlace", OPT_CHOICE(deinterlace,
             {"no", 0}, {"yes", 1}, {"auto", -1})},
+        {"deinterlace-field-parity", OPT_CHOICE(field_parity, 
+            {"tff", 0}, {"bff", 1}, {"auto", -1})},
```

Comment:
ideally, these would use the same defines as you used in the filters. I imagine that means pulling them out of `refqueue.h` for dependency reasons, but it will avoid any danger of divergence.